ipn/ipnlocal/serve.go

```diff
@@ -575,6 +575,20 @@ type reverseProxy struct {
 	h2cTransport lazy.SyncValue[http.RoundTripper]
 }
 
+// closeIdleConnections closes any connections open for either of its transports
+func (rp *reverseProxy) closeIdleConnections() {
```

Comment:
maybe call this `Close` so that its clear that this `reverseProxy` can no longer be used.